src/MetadataUtility/Metadata/FrontierLabs/FlacHeaderExtractor.cs

```diff
@@ -7,28 +7,45 @@ namespace MetadataUtility.Metadata.FrontierLabs
     using System.Threading.Tasks;
     using MetadataUtility.Audio;
     using MetadataUtility.Models;
+    using Microsoft.Extensions.Logging;
     using NodaTime;
 
     public class FlacHeaderExtractor : IMetadataOperation
     {
+        private readonly ILogger<FlacHeaderExtractor> logger;
+
+        public FlacHeaderExtractor(ILogger<FlacHeaderExtractor> logger)
+        {
+            this.logger = logger;
+        }
+
         public ValueTask<bool> CanProcessAsync(TargetInformation information)
         {
-            var result = !information.IsFlacFile();
+            var result = information.IsFlacFile();
 
             return ValueTask.FromResult(result);
```

Comment:
Initially made an assertion for this, but it broke the MetadataCommand test (since the empty files don't have enough bytes for a header). Instead I added this logic into the CanProcessAsync, changing IsFlacFile to IsValidFlacFile, does that work?

Reply:
Also, I verified the stream was at least [42 bytes](https://github.com/QutEcoacoustics/emu/pull/121/files#:~:text=buffer.StartsWith(FlacMagicNumber)%20%26%26-,stream.Length%20%3E%2042,-%3B), which includes everything up to the end of the [metadata black stream info](https://xiph.org/flac/format.html#metadata_block_streaminfo).